export const getConsumoRodeo = async (req, res, next) => {
    try {
    } catch (error) {
        next(error);
    }
};

export const createConsumoRodeo = async (req, res, next) => {
    try {
    } catch (error) {
        next(error);
    }
};

export const getConsumoVaca = async (req, res, next) => {
    try {
    } catch (error) {
        next(error);
    }
};

export const createConsumoVaca = async (req, res, next) => {
    try {
    } catch (error) {
        next(error);
    }
};
